DTML.EduBot/Web.config

```diff
@@ -9,6 +9,9 @@
     <add key="BotId" value="YourBotId" />
     <add key="MicrosoftAppId" value="" />
     <add key="MicrosoftAppPassword" value="" />
+    <add key="tokenEndpoint" value="https://api.cognitive.microsoft.com/sts/v1.0/issueToken"/>
+    <add key= "translatorEndpoint" value = "http://api.microsofttranslator.com/v2/Http.svc/Translate"/>
```

Comment:
This is good. Can you also put the API keys in the config? Before checking in, you can set it as blank value.